frontend/next-config/redirects/part-collections.js

```diff
@@ -14,7 +15,7 @@ function mapPartItemTypes() {
    const legacyTagsWithNoItemType = legacyFilterSlugs['tagsWithNoItemType'];
    // Redirect tags we don't support anymore to the base product list page.
    const legacyToParent = legacyTagsWithNoItemType.map((oldTag) => ({
-      source: `/Parts/:slug/${oldTag}`,
+      source: `/Parts/:slug/${escapeStringForRegex(oldTag)}`,
```

Comment:
next.js only claims to support regexes inside parens after parameters (like the other use in `next.config.js`). Here we have a regex outside parens. They don't say if they split the string on `/` then interpret a regex, nor even if they support regexes outside params:

![image](https://user-images.githubusercontent.com/26855/189227029-d85735e0-f7ec-4974-a94e-fe245efd9eaf.png)

Sounds like this will take experimentation to validate if it works. I'm still quite leery of using an or-list `x|y|z` with no surrounding parens.

Reply:
Good catch, I hadn't noticed that discrepancy. Given that we didn't use parens before, it probably _works_ as is (we don't use an or-list here, that's only for tools). But I'd like to follow the suggested pattern, so I'm gonna add them.

I also noticed that I neglected to escape `legacyTagToItemType` above.

Reply:
Hm after looking again, I noticed that the old tags are just used as strings here. Since they aren't actually an or-list or any other kind of regex, I don't think we should wrap them in parens.